code/game/machinery/vending.dm

```diff
@@ -1045,10 +1045,11 @@
 					/obj/item/ammo_magazine/ammobox/shotgun/beanbags = 10,
 					/obj/item/ammo_magazine/ammobox/shotgun/flashshells = 10,
 					/obj/item/ammo_magazine/ammobox/shotgun/blanks = 10,
+					/obj/item/storage/pouch/holster = 5,
+					/obj/item/storage/pouch/holster/baton = 5,
 					/obj/item/clothing/accessory/holster = 5,
-					/obj/item/clothing/accessory/holster/armpit = 5,
```

Comment:
Give them a higher price instead. Which you did. Don't remove them outright.

Reply:
![image](https://user-images.githubusercontent.com/112724221/191893498-a9a6aa42-178f-4a97-a1a7-d2a7c672cb2a.png)
???
The variation is removed, not the whole accessory.